docs/dg/dev/upgrade-and-migrate/integrate-api-platform.md

```diff
@@ -0,0 +1,333 @@
+        'api_types' => [
+            'storefront',
+        ],
+
+        // Optional: Configure source directories to search for schema files
+        // 'source_directories' => [
+        //     'src/Spryker',
+        //     'src/SprykerFeature',
+        //     'src/Pyz',
+        // ],
+
+        // Optional: Override generated directory
+        // 'generated_dir' => 'src/Generated/Api',
+
+        // Optional: Override cache directory
+        // 'cache_dir' => '%kernel.cache_dir%/api-generator',
+    ]);
+};
+```
+
+### Configure for GlueStorefront
+
+`config/GlueStorefront/packages/spryker_api_platform.php`
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('spryker_api_platform', [
+        'api_types' => [
+            'storefront',
+        ],
+    ]);
+};
+```
+
+### Configure for GlueBackend
+
+`config/GlueBackend/packages/spryker_api_platform.php`
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('spryker_api_platform', [
+        'api_types' => [
+            'backend',
+        ],
+    ]);
+};
+```
+
+## 4. Update Router Configuration
+
+Update the router dependency provider for each application where you want to enable API Platform routes.
+
+### For Glue application
+
+`src/Pyz/Glue/Router/RouterDependencyProvider.php`
+
+```php
+<?php
+
+declare(strict_types = 1);
+
+namespace Pyz\Glue\Router;
+
+use Spryker\Glue\Router\Plugin\Router\SymfonyRouterPlugin;
+use Spryker\Glue\GlueApplication\Plugin\Rest\GlueRouterPlugin;
+use Spryker\Glue\Router\RouterDependencyProvider as SprykerRouterDependencyProvider;
+
+class RouterDependencyProvider extends SprykerRouterDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\RouterExtension\Dependency\Plugin\RouterPluginInterface>
+     */
+    protected function getRouterPlugins(): array
+    {
+        return [
+            new GlueRouterPlugin(), // Existing Glue API router
+            new SymfonyRouterPlugin(), // Add this for API Platform routes
+        ];
+    }
+}
+```
+
+{% info_block infoBox "Router order matters" %}
+
+The order of router plugins matters. The `SymfonyRouterPlugin` should be added after existing router plugins to ensure proper routing priority.
+
+{% endinfo_block %}
+
+## 5. Generate API resources
+
+After configuration, generate your API resources from schema files:
+
+```bash
+# Generate resources for all configured API types in Glue
+docker/sdk glue api:generate
+
+# Generate resources for all configured API types in GlueStorefront
+docker/sdk GLUE_APPLICATION=GLUE_STOREFRONT glue api:generate
```

Comment:
This command doesn't work for me:
`Unknown command "GLUE_APPLICATION=GLUE_STOREFRONT" is requested. `
maybe

Reply:
maybe 
```suggestion
GLUE_APPLICATION=GLUE_STOREFRONT docker/sdk glue api:generate
```